locales/es-ES.yml

```diff
@@ -0,0 +1,216 @@
+es-ES:
+
+  # global, logged out, and auth-related
+  add: Crear
+  add_new: Crear nuevo
+  already_have_account: Tengo una cuenta # signup form cta
+  back: Atrás
+  buy_now: Comprar
+  cancel: Cancelar
+  change_password: Cambiar Contraseña
+  charge_level: Nivel de Batería
+  charged: Cargada
+  close: Cerrar
+  confirm_change_password: Cambiar mi contraseña
+  confirm_password: Confirmar nueva contraseña
+  create_an_account: Crear cuenta de usuario # login form cta
+  days: días
+  delete: Borrar
+  device_id: Identificador de dispositivo
+  edit: Editar
+  email_address: Correo electrónico
+  first_name: Nombre
+  forgot_password: Olvidé mi contraseña # login form link
+  forgot_password_title: Otra vez # heading on /password/new (hint: https://www.google.com/search?q=here+we+go+again+meme)
+  get_started: ¡Empecemos!
+  identify: Identificar
+  keep_me_signed_in: Mantenerme conectado
+  knowledge_base: Base de Conocimiento
+  last_name: Apellido
+  login: Iniciar sesión
+  log_out: Cerrar sesión
+  minimum_characters: Caracteres como mínimo # ex: 8 characters minimum
+  mins: mins # 'minutes' abbreviation
+  new_password: Nueva contraseña
+  password: Contraseña
+  reset_password: Restablecer mi contraseña
+  sign_in: Iniciar sesión # login form cta
+  sign_up: Registrarse # logged out navigation
+  save: Guardar
+  update: Actualizar
+  view: Ver
+  welcome: ¡Hola!
+
+  # views by resource (device, plugins, playlists, account, etc)
+  account:
+    index:
+      title: Cuenta
+      tagline: Gestionar mi cuenta
+      email_address: Correo electrónico
+      email_address_hint: Se usa para conectarse a TRMNL y recibir mensajes.
+      first_name_hint: Usamos tu nombre para hacer que la interfaz de TRMNL sea más cómoda.
+      last_name_hint: Usamos tu apellido para hacer que la interfaz de TRMNL sea más cómoda.
+      password_hint: Para proteger tu cuenta de TRMNL.
+      session: Sesión
+      session_hint: Estás conectado como %{user}
+      time_zone: Zona horaria
+      time_zone_hint: Determina la tasa de refresco y tiempo de suspensión del dispositivo.
+
+    # toast messages
+    update:
+      success: Perfil actualizado correctamente
+  dashboard:
+    index:
+      title: Panel de control
+      last_7_days: Últimos 7 días
+      last_30_days: Últimos 30 días
+      last_90_days: Últimos 90 días
+      today: Hoy
+      yesterday: Ayer
+    health: # partial (widget)
+      bad: Algunos plugins tienen problemas
+      good: Los plugins funcionan correctamente
+      next_steps: Te informaremos si algo cambia
+    news: # partial (widget)
+      title: Noticias
+    playlists: # partial (widget)
+      title: Listas de reproducción
+      cta: Ir a mis listas de reproducción
+      pieces_of_content_displayed: Contenido mostrado
+    plugins: # partial (widget)
+      title: Plugins
+      cta: Ir a Plugins
+      connected: Conectado
+      new_and_popular: Nuevo y Popular
```

Comment:
I've kept the original English capitalisation but similar phrases seem to vary slightly. Please let me know if there's any unwanted inconsistency.